.github/workflows/ci.yml

```diff
@@ -153,11 +153,35 @@ jobs:
           name: wheels-sdist
           path: dist
 
+  tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14", "3.14t"]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
```

Comment:
```suggestion
        uses: actions/setup-python@v6
```

Reply:
You added a PR to automatically update it, and I will merge it later to let it update them automatically.